packages/storybook/package.json

```diff
@@ -34,20 +34,26 @@
     "@storybook/theming": "^7.6.17",
     "@vitejs/plugin-react": "^4.2.1",
     "ast-types": "^0.14.2",
+    "autoprefixer": "^10.4.18",
     "dotenv": "^16.0.3",
     "http-proxy-middleware": "^2.0.6",
     "i18next": "^22.0.1",
     "moment": "^2.30.1",
+    "postcss": "^8.4.35",
     "react": "^18.2.0",
     "react-calendar": "^3.7.0",
-    "react-docgen": "^6.0.1",
     "react-dnd": "^11.1.3",
     "react-dnd-html5-backend": "^11.1.3",
+    "react-docgen": "^6.0.1",
     "react-dom": "^18.2.0",
     "semantic-ui-react": "^2.1.2",
     "simple-keyboard-layouts": "^3.1.87",
     "storybook": "^7.0.26",
+    "tailwindcss": "^3.4.1",
     "underscore": "^1.13.2",
     "vite": "^5.1.4"
+  },
+  "dependencies": {
+    "@headlessui/react": "^1.7.18"
```

Comment:
Let's keep all of the dependencies in `devDepenencies` for consistency.

Reply:
Whoops! Fixed.